okapi-core/pom.xml

```diff
@@ -168,6 +168,11 @@
       <version>4.0.3</version>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>org.folio</groupId>
+      <artifactId>testing</artifactId>
+      <scope>test</scope>
+    </dependency>
```

Comment:
org.folio:testing depends on junit:junit only: https://github.com/folio-org/raml-module-builder/blob/v32.2.0/testing/pom.xml

We cannot move UtilityClassTester.java from org.folio:testing to org.folio:folio-service-tools-test
https://github.com/folio-org/folio-service-tools/blob/v1.6.1/folio-service-tools-test/pom.xml
because org.folio:folio-service-tools-test depends on org.folio:domain-models-runtime.

Reply:
It's possible to make a okapi testing pom and then RMB might use that. But it doesn't have to , immediately. Then the direction is one way.

Reply:
https://issues.folio.org/browse/FOLIO-3052 "Move org.folio:testing into own repository"

Reply:
> > Obviously he/she is using it
> 
> A single person is using a non-documented feature. For me this is not enough to have inconsistent names (underscore versus dot) in okapi.conf (`deploy_waitIterations=`) and on the command line (`-Ddeploy.waitIterations=`).

Remind me again , why `http.port` was seen as better naming than just `port`? We have applied that in a number of places already.

I'm not saying that dots are better than underscores, but I made a judgement call that keeping behavior was better than changing. That's the contract we make with semantic versioning.

If you feel strongly about it (which I don't understand), I suggest you make a PR that accepts both.

I wouldn't spend time on it. There are more important tasks out there.